proto/osmosis/lockup/lock.proto

```diff
@@ -55,3 +55,14 @@ message QueryCondition {
     (gogoproto.moretags) = "yaml:\"timestamp\""
   ];
 }
+
+// SyntheticLock is a single unit of synthetic lockup
+message SyntheticLock {
+  uint64 lock_id = 1;
+  string suffix = 2;
```

Comment:
Is this the name of the synthetic lock denom?

Reply:
Yes, it's what comes after native lockup's denom.

Reply:
Added comment for suffix field on proto definition